src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -674,6 +684,14 @@ public string Description
             }
         }
 
+        /// <summary>
+        /// List of user defined data from extensions and view extensions stored in the graph
+        /// </summary>
+        internal ICollection<ExtensionData> ExtensionData
```

Comment:
This is an `ICollection` because you need to Add to it?

Reply:
yep